test/Writer/ErrorLogTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace ZendTest\Log\Writer;
+
+use Zend\Log\Logger;
+use Zend\Log\Writer\ErrorLog;
+
+class ErrorLogTest extends \PHPUnit_Framework_TestCase
```

Comment:
Add tests for:

- validating an exception is thrown if the mode is out of range during instantiation
- validating an exception is thrown if the destination is not a stream when the mode is 3